lib/app/helpers/gravatar_helper.rb

```diff
@@ -3,8 +3,9 @@ module GravatarHelper
 
     def gravatar_tag(url, options = {})
       size = options.fetch(:size) { 20 }
+      theme = options.fetch(:theme, 'img-circle')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.